site/src/request_handlers/github.rs

```diff
@@ -76,19 +76,48 @@ async fn handle_issue(
     Ok(github::Response)
 }
 
+fn get_awaiting_on_bors_message() -> String {
+    format!(
+        "Awaiting bors try build completion.
+
+@rustbot label: +S-waiting-on-perf
+
+{COMMENT_MARK_TEMPORARY}"
+    )
+}
+
 /// The try request does not have a `sha` or a `parent_sha` but we need to keep a record
 /// of this commit existing. The DB ensures that there is only one non-completed
 /// try benchmark request per `pr`.
 async fn record_try_benchmark_request_without_artifacts(
+    client: &client::Client,
     conn: &dyn database::pool::Connection,
     pr: u32,
     backends: &str,
-) {
+) -> ServerResult<github::Response> {
     let try_request = BenchmarkRequest::create_try_without_artifacts(pr, backends, "");
     log::info!("Inserting try benchmark request {try_request:?}");
-    if let Err(e) = conn.insert_benchmark_request(&try_request).await {
-        log::error!("Failed to insert try benchmark request: {}", e);
+
+    match conn.insert_benchmark_request(&try_request).await {
+        Ok(BenchmarkRequestInsertResult::AlreadyQueued) => {
+            log::error!(
+                "Failed to insert try benchmark request, a request for PR`#{pr}` already exists"
+            );
+            client.post_comment(pr, format!("Failed to insert try benchmark request, a request for PR`#{pr}` already exists")).await;
```

Comment:
```suggestion
            client.post_comment(pr, format!("This pull request was already queued before and is awaiting a try build to finish.")).await;
```
Otherwise it sounds like there was some failure in the bot itself.